docker/Dockerfile

```diff
@@ -172,13 +189,29 @@ COPY "./models" "./models"
 COPY "./scripts" "./scripts"
 COPY ["*.md", "LICENSE", "./"]
 
+USER ${MORPHEUS_USER}
+
 # Use morpheus by default
 CMD [ "morpheus" ]
 
 # ============ Stage: development ============
 # Install and configure development only packages
 FROM conda_env_dev as development
 
+ARG MORPHEUS_USER="root"
+
+# Conditionally create user, install sudo, and allow user to run sudo commands
+RUN if [ ! -z "${MORPHEUS_USER}" ] && [ "${MORPHEUS_USER}" != "root" ] ; then \
+    useradd -ms /workspace ${MORPHEUS_USER} \
+      && usermod -aG sudo ${MORPHEUS_USER} \
+      && apt-get update \
+      && apt-get install -y sudo \
+      && apt-get clean \
+      && echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers ; \
+    else \
+      echo "Skipping user creation..." ;\
+    fi
+
```

Comment:
Can we move this into one of the above layers (such as base or conda_env) to avoid duplicating it in both the `development` and `runtime` targets? We dont need to set the `USER` until needed but we certainly can create the user in the base image.

Reply:
Yes, this makes sense. I'll move it up.